specifyweb/frontend/js_src/lib/localization/query.ts

```diff
@@ -650,28 +650,8 @@ export const queryText = createDictionary({
   importNoReadPermission: {
     'en-us':
       'The query you imported contains tables you do not have read access to:',
-    'es-es': `
```

Comment:
Not sure why this was removed once again

Reply:
If you want, I can add it back myself

Reply:
ok thank you

Reply:
Note, I also renamed the commit message to `Remove unnecessary separator in combo box`

(the commit message should be structured like `This commit will <commit message>` - `will` should not be part of the commit message)

Also `Remove` is better than `Delete` in this context (delete is better suited for files, resources)